pkg/ext-proc/handlers/request.go

```diff
@@ -82,20 +82,29 @@ func (s *Server) HandleRequestBody(
 	if err != nil {
 		return nil, fmt.Errorf("failed to find target pod: %w", err)
 	}
+
 	logger.V(logutil.DEFAULT).Info("Request handled",
 		"model", llmReq.Model, "targetModel", llmReq.ResolvedTargetModel, "endpoint", targetPod)
 
+	// Insert target endpoint to instruct Envoy to route requests to the specified target pod.
```

Comment:
This enables us to not store the port number with the pod in the datastore. The port number is only needed when the epp responds with the endpoint.